lib/beaker/dsl/helpers.rb

```diff
@@ -23,14 +24,12 @@ module DSL
     #
     #
     module Helpers
-      include Beaker::DSL::Helpers::FacterHelpers
       include Beaker::DSL::Helpers::HostHelpers
-      include Beaker::DSL::Helpers::PuppetHelpers
       include Beaker::DSL::Helpers::TestHelpers
-      include Beaker::DSL::Helpers::TKHelpers
```

Comment:
where are the tk-helpers going?  into beaker-puppet? 
or are we just eliminating the auto-include in this vesion for tkhelpers and facter?